STEERING-COMMITTEE.md

```diff
@@ -7,51 +7,55 @@ aliases:
   - /contributing/steering-committee/
 ---
 
-The Knative Steering Committee (KSC) is the ultimate authority for the Knative
-project, and governs all aspects of the project.
+The Knative Steering Committee (KSC) defines the governance for Knative
+in pursuit of the goal of an open, vibrant project with a productive
+community, a healthy ecosystem of vendors, and most of all satisfied
+customers running their workloads on Knative clusters.
 
 The governance of Knative is an open, living document, and will continue to
 evolve as the community and project change. We expect over time we will adapt
 the way we run this committee, based on feedback from the community.
 
+- [Mission](#mission)
 - [Charter](#charter)
 - [Delegated authority](#delegated-authority)
 - [Committee Meetings](#committee-meetings)
 - [Committee Mechanics](#committee-mechanics)
 - [Committee Members](#committee-members)
-  - [Allocation of seats](#allocation-of-seats)
+  - [Composition](#composition)
+  - [Nomination of representatives](#nomination-of-representatives)
 - [Decision process](#decision-process)
 - [Changes to the charter](#changes-to-the-charter)
 - [Getting in touch](#getting-in-touch)
 
+## Mission
+
+The Steering Committee's mission is to guide the project such that:
+
+1. Knative is a truly open effort which is inclusive and empowers contributors
```

Comment:
I would remove truly because again that is subjective, otherwise you would have to define what a truly open effort is.

Reply:
+1 (re: `truly`), but I don't think "open effort" is any more clear.  Open source is a spectrum and it's not clear where "effort" registers on that spectrum.

Perhaps @anicolao meant "open project", which would indicate more than simply "open _source_ project" (thus dropping the qualification "source") and would typically include "[open governance](http://opengovernance.dev/)", but that is not what this proposal outlines (#214 does, or at least gets us close)